traitsui/testing/tester/_ui_tester_registry/wx/_traitsui/ui_base.py

```diff
@@ -9,7 +9,8 @@
 #  Thanks for using Enthought open source!
 #
 from traitsui.wx.ui_base import ButtonEditor
-from traitsui.testing.api import DisplayedText, IsEnabled, MouseClick
+from traitsui.testing.tester.command import MouseClick
+from traitsui.testing.tester.locator import DisplayedText, IsEnabled
```

Comment:
This should be `traitsui.testing.tester.query`, not `traitsui.testing.tester.locator`.
(Wx CI job is still allowed to fail, hence the green tick, but a lot of tests are failing due to this import.)

Reply:
Oops, nice catch thank you!